frontend/src/components/categories/dropdown.tsx

```diff
@@ -110,7 +107,7 @@ export const Location = () => {
         background="var(--deep-charcoal)"
         border="1px solid var(--bg-two)"
       >
-        {locations?.map((location, index) => {
+        {locations?.map((location: string, index: any) => {
```

Comment:
Let's allow `index` to receive a type of `number`

Reply:
Alright. I'm on it

Reply:
Thanks chief!!🙌🏽

Reply:
I've done that.